content/backmatter.tex

```diff
@@ -143,66 +156,62 @@ \section{Running Programs}
 
 
 
-
 \chapter{Undefined Behavior in OpenSHMEM}\label{sec:undefined}
 
-The specification provides guidelines to the expected behavior of
-various library routines.  In cases where routines are improperly used
+The specification formalizes the expected behavior of
+its library routines.  In cases where routines are improperly used
 or the input is not in accordance with the specification, undefined
-behavior may be observed.  Depending on the implementation there are
+behavior is observed.  Depending on the implementation, there are
 many interpretations of undefined behavior. 
 
-$\;$
-
-$ $%
-\begin{tabular}{|>{\raggedright}p{0.3\textwidth}|>{\raggedright}p{0.6\textwidth}|}
+\begin{longtable}{|>{\raggedright}p{0.3\textwidth}|>{\raggedright}p{0.6\textwidth}|}
 \hline 
 \textbf{Inappropriate Usage} & \textbf{Undefined Behavior}\tabularnewline
 \hline 
-\hline 
-Uninitialized library & If \openshmem is not initialized through a call to
-\FUNC{shmem\_init}, subsequent accesses to \openshmem routines have undefined
-results.  An implementation may choose, for example, to try to continue or abort
-immediately upon the first call to an uninitialized routine.\tabularnewline
-\hline 
+\endhead
+Uninitialized library & If the \openshmem library is not initialized,
+calls to non-initializing \openshmem routines have undefined
+behavior.  For example, an implementation may try to continue or may abort
+immediately upon an \openshmem call into the uninitialized library.
+\tabularnewline
+\hline
+Multiple calls to initialization routines & In an OpenSHMEM program where
+an initialization routine (\FUNC{shmem\_init} or \FUNC{shmem\_init\_thread})
+has already been called, any subsequent calls to an initialization routine
```

Comment:
IIRC, start_pes can be called multiple times.  Replace "an initialization routine" with "these routines"?